NSS-Day-Cohort-49/tabloid-cli-lionfish
# Ar tab blog

Modified file BlogRepository.cs & BlogManager.cs
​
Steps to Review
  -Checkout this branch locally.
  -git fetch --all
  -git checkout ar-tabBlog
  -Open a new Terminal tab and navigate to the server directory.
  -Test app functionality.
  -Instructions for how reviewer can test functionality, and detailed description of what the expected outcome is:
  Example: When user does goes to the events page the should be able to see the weather forecast for that event if the event date is in the range of the returned weather data.

- View code file.
- Confirm file modifications are present as indicated above.
  
- Test regressive code.
